marlowe-dashboard-client/src/Contract/State.purs

```diff
@@ -231,9 +225,10 @@ handleAction ::
   ManageMarloweStorage m =>
   Toast m =>
   Input -> Action -> HalogenM State Action ChildSlots Msg m Unit
-handleAction _ (SetNickname nickname) = do
+handleAction { followerAppId } SelectSelf = callMainFrameAction $ MainFrame.DashboardAction $ Dashboard.SelectContract $ Just followerAppId
+
+handleAction { followerAppId } (SetNickname nickname) = do
```

Comment:
We are not using the app id here right?
```purescript
handleAction _ (SetNickname nickname)
```

Reply:
yes, we need it to `insertIntoContractNicknames` (contractNicknames is a `Map PlutusAppId String`)